app/assets/stylesheets/local/typography.scss

```diff
@@ -187,13 +187,12 @@ h1.heading-start {
   line-height: 1.25;
   margin-top: 0.625em;
   margin-bottom: 0.3125em;
-  
```

Comment:
I would add the space that was here back in too :) (again just a small thing for readability)